src/intpack.f90

```diff
@@ -1413,9 +1410,8 @@ subroutine opaa0(l,m,n,ga,v,d)
       itot=l+m+n
       itoth=itot/2
       pre=(2.d0*pi/ga)*fn(l+1)*fn(m+1)*fn(n+1)
-      if(2*itoth-itot) 1,2,1
-   1  pre=-pre
-   2  del=.25d0/ga
+      pre=-pre
+      del=.25d0/ga
```

Comment:
This part implements a multiplication with `-1**(itot)` using an arithmetic if and integer division: The condition in the arithmetic if should be replaced by `2*itoth-itot` -> `2*(itot/2)-itot` -> `-modulo(itot, 2)`, therefore the correct refactor would be

```f90
if (modulo(itot, 2) == 1) pre = -pre
```